assets/out

```diff
@@ -92,6 +98,13 @@ push_tags() {
   git push --tags push-target $forceflag
 }
 
+add_and_push_notes() {
+  if [ -n "$notes" ]; then
+    git notes add -m "${notes}"
```

Comment:
This would be cleaner as `git nodes add -F $notes_file` rather than `cat`ing it into a variable